universal-login-relayer/lib/services/messages/PendingMessages.ts

```diff
@@ -1,22 +1,21 @@
-import {Wallet, Contract, providers} from 'ethers';
+import {Wallet, Contract} from 'ethers';
 import {calculateMessageHash, concatenateSignatures, SignedMessage, INVALID_KEY, ensure, MessageStatus} from '@universal-login/commons';
 import WalletContract from '@universal-login/contracts/build/WalletMaster.json';
 import {DuplicatedSignature, InvalidSignature, DuplicatedExecution, InvalidTransaction, NotEnoughSignatures} from '../../utils/errors';
 import IPendingMessagesStore, {CollectedSignatureKeyPair} from './IPendingMessagesStore';
 import {getKeyFromHashAndSignature, sortSignatureKeyPairsByKey, createPendingMessage} from '../../utils/utils';
-
-declare type DoExecute = (message: SignedMessage) => Promise<providers.TransactionResponse>;
+import MessageQueueService from './MessageQueueService';
 
 export default class PendingMessages {
 
-  constructor(private wallet : Wallet, private messagesStore: IPendingMessagesStore, public doExecute: DoExecute) {
+  constructor(private wallet : Wallet, private messagesStore: IPendingMessagesStore, public messageQueue: MessageQueueService) {
   }
 
   async isPresent(messageHash : string) {
     return this.messagesStore.isPresent(messageHash);
   }
 
-  async add(message: SignedMessage) : Promise<providers.TransactionResponse | MessageStatus> {
+  async add(message: SignedMessage) : Promise<string | MessageStatus> {
```

Comment:
Should be `Promise<MessageStatus>`, why two different types?

Reply:
Message doesn't have field 'id' and shouldn't have. Now we need 'id', but in future single store for Messages will have only messageHash without id. It was simpler change i guess.

Reply:
Changed ✔️